changelogs/unreleased/5349-cleanup-old-agents.yml

```
@@ -0,0 +1,7 @@
+---
+description: cleanup old agents from the agent table in db
```

I would add a capital letter and a dot so that it looks better in the final changelog, and maybe rephrase it slightly: (not a strong opinion) 
```suggestion
description: Old agents in the agent table are now cleaned up from the database.
```